### DISCUSSION TITLE
Does this Update Automatically on Start???

### DISCUSSION THREAD

**Discussion by EddieSoloProjects** [Stability-AI/StableSwarmUI#329] [Q&A] [answered]
Do I need to run the Update Bat File or does this Auto Update??

**Comment by mcmonkey4eva** [comment, marked as answer]
Run the update bat when you want to update, Swarm does not update automatically by default.
Swarm _does_ update the comfy backend by default (configurable via Server -> Backends -> edit the backend, there's a checkbox for it)
Swarm does check for updates by default (as of yesterday lol) (configurable via Server -> Server Configuration), this is just for major version number updates and just shows a notice rather than applying the update.

**Reply by EddieSoloProjects** [reply]
Thanks for the NFO :)

**Comment by dog0311** [comment]
I tried to follow the update on the server page, it stayed over night saying it was open. I closed edge and it stayed on "[Warning] User local requested update-and-restart." in the terminal window. So I am not sure what to do to update.  Also is there a way to only show the packages that my pc can handle I have an asus z790 Gaming Wifi7 mb i9 cpu 64gb Ram and an RTX4080 Super 16gb VRAM.

I am going to try and reboot, open the terminal windows again and run the launch-window.bat again and see if it does a self update. Wait I'll open the .bat up and see if in has a place to pass an update arg.

**Reply by dog0311** [reply]
Negative on the update arg.
